pkgs/by-name/ld/ld64/patches/0006-Add-Meson-build-system.patch

```diff
@@ -299,7 +299,7 @@ index 0000000..067a69c
 +    install : true,
 +    sources : [
 +        configure_h,
-+        'UnwindDump.cpp',
++        'unwinddump.cpp',
```

Comment:
We should probably PR that to @reckenrode so that it isn't undone the next time the patches are exported https://github.com/reckenrode/ld64/blob/ld64-951.9-nixpkgs/src/other/meson.build#L72

Reply:
I asked him on Matrix at the time and he [asked me to do it in Nixpkgs](https://matrix.to/#/!lheuhImcToQZYTQTuI:nixos.org/$e-iaXPjZ0monjmPrfjQ9NhwPBtewvKFuLvIdyPhEm6U?via=nixos.org&via=matrix.org&via=nixos.dev) so that he could fold the changes back in on next rebase. Since he rebases the patch stack for new `ld64` versions (for which there is one pending), a GitHub PR wouldn’t quite work.